logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/ProtoContentMapper.kt

```diff
@@ -49,11 +60,23 @@ class ProtoContentMapperImpl: ProtoContentMapper {
             is GenericMessage.Content.Composite -> MessageContent.Unknown
             is GenericMessage.Content.Confirmation -> MessageContent.Unknown
             is GenericMessage.Content.DataTransfer -> MessageContent.Unknown
-            is GenericMessage.Content.Deleted -> MessageContent.Unknown
+            is GenericMessage.Content.Deleted -> MessageContent.DeleteMessage(genericMessage.messageId)
             is GenericMessage.Content.Edited -> MessageContent.Unknown
             is GenericMessage.Content.Ephemeral -> MessageContent.Unknown
             is GenericMessage.Content.External -> MessageContent.Unknown
-            is GenericMessage.Content.Hidden -> MessageContent.Unknown
+            is GenericMessage.Content.Hidden -> {
+                val hiddenMessage = genericMessage.hidden
+                if (hiddenMessage != null) {
+                    MessageContent.DeleteForMe(
+                        hiddenMessage.messageId,
+                        ConversationId(hiddenMessage.qualifiedConversationId!!.id,
```

Comment:
Missed this. We need to remove the force unwrap here and gracefully ignore the message if it's null.